src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -190,7 +190,7 @@ const NavHeader = (props) => {
               style={[
                 styles.domain(isHeaderHide),
                 {
-                  bottom: normalize(isFollow && followType === 'signed' ? 13 : 24),
+                  bottom: normalize(isFollow ? 13 : 24),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change removes the condition `followType === 'signed'` from the ternary operator. This could potentially alter the behavior of your application if there are other possible values for `followType` that should not result in a `bottom` value of `13`. If `followType` only has a binary state (either it's `'signed'` or not), then this change is fine. However, if there are more states, you might want to reconsider this change.
```diff
- bottom: normalize(isFollow && followType === 'signed' ? 13 : 24),
+ bottom: normalize(isFollow ? 13 : 24),
```
Please ensure that this change aligns with the intended logic and doesn't introduce unexpected behavior.


<!-- This is an auto-generated comment by OSS CodeRabbit -->